backend/src/gateway/controllers/index.ts

```
@@ -0,0 +1,91 @@
+import {Request, Response} from 'express'
+import GroupChats from '../../models/Groupchat'
+import groupchat from '../../types/groupchat'
+import Messaging from '../../models/Messages'
+import User from '../../models/User'
+import user from '../../types/user'
+
+
+async function getGroupChatByID(req: Request, res: Response){
+    try {
+        const gcID = req.query.id
+        const groupChat = await GroupChats.findById({"_id": gcID})
+        if(groupChat === null){
+            res.status(200).json({"message":"GROUPCHAT DOES NOT EXIST"})

```

if group chat does not exist it should return a 404 error not a 200 error